packages/pointer_interceptor/pointer_interceptor_web/pubspec.yaml

```
@@ -0,0 +1,31 @@
+name: pointer_interceptor_web
+description: Web implementation of the pointer_interceptor plugin.
+repository: https://github.com/flutter/packages/tree/main/packages/pointer_interceptor/pointer_interceptor_web
+issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen++label%3A%22p%3A+pointer_interceptor%22
+version: 2.3.1
```

This version mismatches the one in the changelog and will break the build.

```suggestion
version: 0.10.0
```